docs/tutorials/frontier/index.md

```diff
@@ -0,0 +1,132 @@
+included in Frontier. This can be done starting at **any commit within frontier** to get a specific version/tag.
+
+```bash
+# from the top working dir of Frontier:
+cd .maintain/
+# set the *full file name with .tar.gz extension* for your output file
+./node-template-release.sh TEMPLATE.tar.gz
+# Note the file will be placed in the top level working dir of frontier
+# Move the archive to wherever you like...
+tar xvzf TEMPLATE.tar.gz
+# this unpacks into `frontier-node-template` with all your files
+cd frontier-node-template
+# build the template
+cargo build --release
+```
+
+### Github Template
+
+The Substrate Developer Hub has generated the template using the
+[included release guide](https://github.com/paritytech/frontier/blob/master/docs/node-template-release.md)
+, and intends to update with major releases of Frontier moving forward.
+
+You can `use` the pre-generated template or `fork` it from here:
+https://github.com/substrate-developer-hub/frontier-node-template/ .
+
+### Testing Ethereum Functionality
+
+To test the template, please see the [included
+README](https://github.com/paritytech/frontier/blob/master/template/README.md) that details the
+proper Ethereum account use, as well as deploying and interacting with
+EVM contracts.
+
+## Frontier Overview & Architecture Diagrams
+
+Here are a few helpful diagrams to help illustrate how the Frontier EVM and Ethereum RPC plug into
+your Substrate FRAME runtime.
+
+### EVM Pallet Runtime Configuration
+
+The Ethereum Virtual Machine (EVM) is a sandboxed virtual stack machine that is implemented in the
+EVM pallet. The EVM is responsible for executing Ethereum contract bytecode of smart contracts,
+typically written in a high level language like Solidity, then compiled to EVM bytecode.
+
+![architecture diagram](assets/tutorials/frontier/pallet-evm.png)
+
+### Ethereum Pallet
+
+The Ethereum pallet is responsible for storing Ethereum-formatted blocks, transaction receipts, and transaction statuses.
+
+The biggest challenge here is that the Frontier project does not use the same, published, Substrate
+crates; it takes Substrate code from github. For this tutorial, I've made a special branch of
+Frontier that uses the published dependencies.
+
+![architecture diagram](assets/tutorials/frontier/pallet-ethereum.png)
+
+### Wrapping Ethereum Transactions
+
+When a user submits a raw Ethereum transaction, we need to convert it into a Substrate transaction. The conversion is simple. We just wrap the raw transaction in a call the `pallet_ethereum`'s `transact` extrinsic. This is done in the runtime.
+
+> Note that Ethereum Accounts and Substrate accounts in this template are not directly compatible
+> for using keys. For an explainer on this, please see the
+> [Moonbean documentain on EVM&Substrate Accounts](https://docs.moonbeam.network/learn/unified-accounts/#substrate-evm-compatible-blockchain)
```

Comment:
What's the purpose of this section? As the learner, what am I expected to do here?

Reply:
#1085 to include.